contracts/SnappAuction.sol

```diff
@@ -30,21 +30,15 @@ contract SnappAuction is SnappBase {
 
     event SellOrder(
         uint auctionId,
-        uint16 slotIndex,
+        uint16 slotStartIndex,
         uint16 accountId,
-        uint8 buyToken,
-        uint8 sellToken,
-        uint96 buyAmount,
-        uint96 sellAmount
+        bytes packedOrder
```

Comment:
Not sure what happened to my comment from earlier, but do we really want this change? this will require a bunch more changes in the listener, and if at all necessary it should maybe happen in a separate PR?

Reply:
I don't understand the problem here! Is it the plural vs non-plural?

Reply:
@fleupold This whole Pr was about changing the sellorderEvent, wasn't it? Do you think having a seperate Event would be better?

Reply:
I was under the impression the whole PR was about changing the *StandingOrderEvent*. The sellOrder event doesn't necessarily need to be adjusted.